### DISCUSSION TITLE
What's the meaning of results.masks.segments when task is segment?

### DISCUSSION THREAD

**Discussion by dbcSep03** [ultralytics/ultralytics#1105] [Q&A]
When task is segment,results.masks.segments is the list of two decimals.
What's the meaning of two decimals?
Thank you for your answer.
![image](https://user-images.githubusercontent.com/95664996/220662004-e28bf10d-9df1-4517-8672-8dd270152c9e.png)

**Comment by Laughing-q** [comment]
@dbcSep03 it's the bounding coordinates of predicted masks but normalized by img-size(w, h). Each row is the coordinate of [x, y].
